Backpacks.cs

```diff
@@ -581,6 +591,38 @@ private bool VerifyCanOpenBackpack(BasePlayer looter, ulong ownerId)
             return true;
         }
 
+        void OnPlayerConnected(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void OnPlayerSleepEnded(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void CreateGUI(BasePlayer player)
+        {
+            if (player == null || player is NPCPlayer || player is HTNPlayer)
+                return;
+
+            if (!_instance._config.GUI.Enabled || !permission.UserHasPermission(player.UserIDString, UsagePermission))
+                return;
+
+            var panelName = "BackpacksUI";
+            CuiHelper.DestroyUi(player, panelName);
+            var elements = new CuiElementContainer();
+            var BackpacksUIPanel = elements.Add(new CuiPanel { Image = { Color = $"{_instance._config.GUI.Color}" }, RectTransform = { AnchorMin = $"{_instance._config.GUI.AnchorMin}", AnchorMax = $"{_instance._config.GUI.AnchorMax}" }, CursorEnabled = false }, "Overlay", "BackpacksUI");
+            elements.Add(new CuiElement { Parent = "BackpacksUI", Components = { new CuiRawImageComponent { Url = $"{_instance._config.GUI.Image}" }, new CuiRectTransformComponent { AnchorMin = "0 0", AnchorMax = "1 1" } } });
+            elements.Add(new CuiButton { Button = { Command = "backpack.open", Color = "0 0 0 0" }, RectTransform = { AnchorMin = "0 0", AnchorMax = "1 1" }, Text = { Text = "" } }, BackpacksUIPanel);
```

Comment:
This is hard to read, can you format multiline?

Also, some of these values are using string interpolation syntax but can just be referring to the variables.